demodata/recipe1.yaml

```diff
@@ -2,10 +2,11 @@ fixtures:
   environment:
     PLANET: Earth
 
-  git_url: https://github.com/RedHatQE/newa.git
-  git_ref: main
-  tmt_path: demodata
-  plan: /plan1
+  tmt:
+    url: https://github.com/RedHatQE/newa.git
+    ref: main
```

Comment:
I am wondering if we should not rather keep follow cli paramets naming (ie. {url,ref} -> git_{url,ref}). If would safe one "mind-step" from the recipe to cli parameters.

Reply:
I am still trying to follow idea that one day newa could run TMT command instead od using Testing Farm and from this perspective I find these params a bit more generic.. Also translation to TF API would look differently I think. But I May be convinced doing otherwise.

Reply:
Discussed in person. Testing-farm API actually contains tmt sections with tmt options and only translates things like --git-url or --git-ref to their tmt equivalents. Hence I am really happy with the current implementation in the end. Sorry for false alarms :).